docs/index.md

```diff
@@ -41,7 +41,7 @@ From this page, you can generate a new API token, and then copy it to use for th
 ## Example Usage
 
 ```terraform
-provider "tmc" {
+provider "tanzu-mission-control" {
   endpoint = var.endpoint # optionally use TMC_ENDPOINT env var
   token = var.token # optionally use TMC_CSP_TOKEN env var
```

Comment:
update the token to reflect the new attribute key value

Reply:
This is as part of PR https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/28